src/app/views/main/main.component.scss

```diff
@@ -31,9 +31,21 @@
             position: absolute;
             right: 0;
             margin: 1rem;
+            display: flex;
+            width: 15rem;
+            justify-content: space-between;
```

Comment:
Use padding on the elements instead of this property! i.e. .search-criteria { padding-right: 1rem } etc.

Reply:
instead of display: flex?

Reply:
Instead of justify-content. It will give problems when adding additional elements